### ISSUE TITLE
Cannot send multipart response from aiohttp server with unbounded range for streaming

### ISSUE THREAD

**Issue by wise0wl** [aio-libs/aiohttp#2946]
## tl;dr

StreamResponse seems unable to send data without being chunked.  

## My specific situation

My application is writing and appending mp4 video data to disk and needs serve this video file as it is being written and continue until a timeout is reached and EOF is reached.  A normal Response object will not do, due to the requirement of having a fully complete response body.  StreamResponse would *seem* to be the logical response class.

However, the StreamResponse sets the Transfer-Encoding response header to chunked, even though the StreamResponse.chunked variable returns False. 

## Expected behavior

If chunked options is false, and perhaps the request being prepared includes a Range request chunked encoding should be turned off.  Another option include having a MultipartWriter sub-class in the response that allows for the response to be written in true streaming fashion with boundary handling.  Issuing these requests in a context would make sense.

## Actual behaviour

StreamResponse seemingly always sets Transfer-Encoding to chunked.

## Environment

AioHTTP Version: 3.1.3
Python Version: 3.6.3
Arch: ARMv7
Virtual ENV

**Comment by asvetlov** [comment]
I don't follow what is your problem.
Chunked content is the first class citizen in HTTP world, aiohttp multipart supports writing chunk by chunk as well: https://docs.aiohttp.org/en/stable/multipart.html#sending-multipart-requests

**Comment by hubo1016** [comment]
There are only three legal ways to send a response in HTTP:

1. With a content length
2. Chunked encoding
3. Old HTTP/1.0 behavior: send response without transfer encoding and content length, close connection when EOF

Aiohttp does not support 3, so it always uses chunked encoding when the content length is undetermined.

Multipart data itself is not a valid transfer encoding. Although the length can be determined with boundaries for this type of data, HTTP never tried to determine the content length from content.

**Comment by hubo1016** [comment]
What's the problem of chunked encoding in your situation? It should be transparent to your application.

**Comment by asvetlov** [comment]
Closing the issue, multipart works with chunked encoding.
